packages/backend/src/core/discovery/utils/diffContracts.test.ts

```
@@ -45,7 +45,7 @@ describe(diffContracts.name, () => {
         E: 'ignoreMePlease',
       },
     }
-    const ignoreInWatchMode = ['E']
+    const ignoreInWatchMode = ['random-key', 'E']
```

don't we want to warn (or throw an error)  when unknown key is ignored here?